.github/workflows/ci.yml

```diff
@@ -38,6 +38,8 @@ jobs:
 
       - name: Run Clippy
         uses: actions-rs/cargo@v1
+        env:
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
```

Comment:
yes, without `SENTRY_DSN` env variable, if you run `cargo clippy` with `--all-features` it should fail with the following error:

```
environment variable `SENTRY_DSN` not defined
```

due to this: `env!("SENTRY_DSN").to_string()`

Reply:
Thats bad, because it means it will fail for *everyone* (who hasn’t set that env var).

Reply:
only for everyone that wants to compile it with `profiling` feature (which I've added in one of the last commit).

If you don't use the profiling feature then no error at all.

Honestly this is the best trade-off I could find to still have the `DSN` in the binary without exposing it as clear text on Github.

Reply:
DSNs are not sensitive data. They are shipped longside, or embedded in your executable anyway.

Reply:
ok, so is it safe to hardcode it directly?

@kamilogorek could you confirm I can do that in `sentry-cli`?

Reply:
I'm fine putting DSN here, however, we need to ask around to create a permanent project for that, and not something temporary/just for testing, as it'll be embedded here forever (for that version).

Reply:
@kamilogorek I just had a quick slack chat with Arpad and it seems we're fine passing it as an env instead of hardcoding it in the code.

> we need to ask around to create a permanent project for that

isn't the `sentry-cli` under the `Sentry` organization just that?

Reply:
It is, just wanted to double-check.